### PR TITLE
Azure DevOps pipeline template

### PR DESCRIPTION
**Pull request by dennis-behm**
This is providing a `azure-devops.yaml` definition to supper the proposed git workflow, leveraging the common backend scripts.

### CODE REVIEW

**Comment by M-DLB** [comment]
Thanks for adding the pipeline definition for Azure DevOps.
Since it's using the wazideploy-evidence.sh script from the Common-Backend-Scripts, it would make sense to also add this script (and the revised pipelineBackend.config file) too. It could be another PR.

**Comment by dennis-behm** [comment]
We noticed that subsequent steps were executed - this need to be addressed.

image

**Review by M-DLB — COMMENTED** [review]
One comment about a pipeline variable

**Comment by dennis-behm** [comment]
> One comment about a pipeline variable: should the variable `sshZosKnownHost` be renamed `zosSSHKnownHost` for consistency with the other variables?

FYI @M-DLB - Done.

File: Templates/AzureDevOpsPipeline/azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+# Agent pool where the pipeline runs
+pool:
+  name: $(agentPool)
+
+# Variables used in the pipeline
+variables:
+  ### Variables to be created in Azure pipeline
+  # ) agentPool                            # Agent pool name for Azure Agents to connect to MVS
+  # ) zosSSHConnection                     # zOS - SSH connection name
+  # ) pipelineWorkspace                    # Root directory on z/OS Unix System services to perform builds. E.g. `/u/ado/workspace`
```
Comment (M-DLB) [inline Templates/AzureDevOpsPipeline/azure-pipelines.yml:32]: Can we add a definition for the zosSSHKnownHost?

File: Templates/AzureDevOpsPipeline/README.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+### Required pipeline variables
+
+Variable | Description
+--- | ---
+  agentPool                            | Agent pool name for Azure Agents to connect to MVS
+  zosSSHConnection                     | zOS - The name of the Azure SSH connection name for connecting to z/OS Unix System Services of the LPAR where the build actions will be performed.
+  zosSSHKnownHost                      | The known host entry for secure shell connections. [See Notes](#obtaining-the-known-host-entry-for-secure-shell-connections).
+  pipelineWorkspace                    | Root directory on z/OS Unix System services to perform builds. E.g. `/u/ado/workspace`
+  wdEnvironmentFile                    | The Wazi Deploy environment file for the deployment into the lowest stage
+  zosSFTPHostname                      | zOS - Host name (or Host IP address) for SFTP connection
```
Comment (M-DLB) [inline Templates/AzureDevOpsPipeline/README.md:50]: I think the variable is called `zosHostname` instead of `zosSFTPHostname`.

**Review by M-DLB — COMMENTED** [review]
One last few comments, sorry!

**Review by M-DLB — APPROVED** [review]
All good, thank you @dennis-behm !

**Pull request merged**